types/chart.js/index.d.ts

```diff
@@ -40,6 +40,7 @@ declare class Chart {
 }
 declare class PluginServiceStatic {
     register(plugin?: PluginServiceRegistrationOptions): void;
+    unregister(plugin?: PluginServiceRegistrationOptions): void;
```

Comment:
Could you please remove the `?` here? I don’t think it makes any sense to make this argument optional; it’s an error in the existing `register()` declaration.

Reply:
sure, should I remove the `?` from the register method as well?